skymp5-server/cpp/server_guest_lib/AnimationSystem.cpp

```diff
@@ -224,3 +230,58 @@ void AnimationSystem::InitAnimationCallbacks(bool isSweetpie)
                               additionalCallbacks.end());
   }
 }
+
+std::vector<uint32_t> AnimationSystem::GetWeaponKeywordFormIds(
+  uint32_t baseId) const
+{
+  auto& espmBrowser = worldState.GetEspm().GetBrowser();
+  return espmBrowser.LookupById(baseId).rec->GetKeywordIds(
```

Comment:
Theoretically, rec can be nullptr here.
Let's check that and return empty vector in this case.
And write to `spdlog::error`